changelog.md

```diff
@@ -49,6 +49,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   `mudensity` module.
 - `compare_mosaiq_fields` moved from the `msq` module into the `plancompare`
   module.
+- The DICOM coordinate extraction functions - `extract_dicom_patient_xyz()`,
+  `extract_iec_patient_xyz()` and `extract_iec_fixed_xyz()` - have been
+  combined into a single function called `xyz_from_dataset()`. The x, y, z axes
+  can still be returned in either the DICOM, IEC FIXED or IEX PATIENT
+  coordinate systems by passing `coord_system='D'` (`F` or `P`) to the
```

Comment:
So that people reading this API in the wild know what the code is doing I think these parameters values should be as descriptive as is normally expected from function names. See:

https://docs.scipy.org/doc/scipy-0.18.1/reference/optimize.html

Parameter names such as coord_system='dicom', 'iec_fixed', 'iec_patient' makes sense...

Reply:
Just had a look at the function and I see you do have the option for more descriptive naming :)

Reply:
Yeah, I was just a little lazy with the changelog. I'll update to match the docstring better :)